Configuration/src/ConfigurationProviders/config/workload.yaml

```diff
@@ -32,10 +32,10 @@ spec:
     ref:
       apiVersion: services.apps.tanzu.vmware.com/v1alpha1
       kind: ResourceClaim
-      name: my-config  
+      name: sample-config-server
```

Comment:
This ResourceClaim is for Application Configuration Service, not config server, so this name could be confusing